packages/botonic-react/src/components/multichannel/multichannel-utils.js

```diff
@@ -8,8 +8,10 @@ import { Providers } from '@botonic/core'
 export const MULTICHANNEL_WHATSAPP_PROPS = { markdown: false }
 
 export const WHATSAPP_MAX_BUTTONS = 3
+export const WHATSAPP_LIST_MAX_BUTTONS = 10
 export const WHATSAPP_MAX_BUTTON_CHARS = 20
 export const DEFAULT_WHATSAPP_MAX_BUTTON_SEPARATOR = 'More options:'
+export const MENU_BUTOON_WHATSAPP_BUTTON_LIST = 'Menu'
```

Comment:
Isn't it better to set the default text as `Show options`?

Reply:
ok